purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesOrchestrator.kt

```diff
@@ -237,6 +247,19 @@ internal class PurchasesOrchestrator(
         }
     }
 
+    fun handleDeepLink(intent: Intent): Boolean {
+        val deepLink = intent.data?.let { deepLinkHandler.parseDeepLink(it) } ?: return false
+
+        when (deepLink) {
+            is DeepLinkHandler.DeepLink.RedeemRCBPurchase -> {
+                rcBillingPurchaseRedemptionHelper.handleRedeemRCBPurchase(deepLink)
+            }
+        }
+        // WIP: This currently returns before the operation is completed. We might want to provide a way for devs
+        // to detect when the operation is started/completed.
```

Comment:
We can probably take inspiration from the promoted purchases API in iOS: https://revenuecat.github.io/purchases-ios-docs/5.6.0/documentation/revenuecat/purchasesdelegate/purchases(_:readyforpromotedproduct:purchase:).

This allows the developer to control whether to perform the redemption and to get a result back.